src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -17,9 +16,11 @@ export interface ErrorBars {
     }
 }
 
-export type ChartDataSetsWithErrors =  ChartDataset & {
-    errorBars?: ErrorBars
-}
+type ChartDataSetsWithErrors<Data = BarChartDefaultData> = BarChartDataset<Data> & { errorBars?: ErrorBars }
```

Comment:
this is basically the same as before but now specifies that the `ChartType` is `bar`